Makefile

```diff
@@ -92,6 +75,9 @@ INSTALL_DIR = $(INSTALL) -d
 INSTALL_PROGRAM = $(INSTALL) -m755
 INSTALL_DATA = $(INSTALL) -m644
 
+# Toolchain
+MSBUILD = msbuild -verbosity:m
```

Comment:
What do you think about adding `-nologo` here? Those three lines add up to a lot of noise.

Reply:
Done. I also tweaked the PostProcess exe to remove another line for each exe.